app/gui/src/presenter/graph/state.rs

```diff
@@ -742,6 +741,31 @@ impl<'a> ViewChange<'a> {
             None
         }
     }
+
+    /// Figure out if a port expression change is necessary and return node AST id. Returns `None`
+    /// if no changes to the expression are needed.
```

Comment:
Docs should semantically describe what the function does. "Figure out" is a really high-level description. What it really does? When the changes are "not needed"?